src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Lottery lottery = new Lottery();
```

Comment:
You dont need to create Lottery for each .getRandomBall. Let's say you play poker with your friends, with such approach, you don't want deck of cards to be reused for the second game, or third. You want to buy ned deck for each game.